.github/compatibility-test-matrices/release-v8.0.x/genesis-chain-a.json

```diff
@@ -0,0 +1,17 @@
+{
+  "chain-a": [
+    "release-v8.0.x"
+  ],
+  "chain-b": [
+    "release-v8.0.x"
+  ],
+  "entrypoint": [
+    "TestGenesisTestSuite"
+  ],
+  "test": [
+    "TestIBCGenesis"
```

Comment:
I added this file with this test for v8.0.x. I can added for earlier releases, if you think it's worth.

Reply:
I think there's no harm adding this to all the active release lines actually. This test has the potential to catch some pretty nasty bugs.